packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.java

```diff
@@ -32,6 +33,9 @@
   private boolean mIsAuthenticating = false;
   private ModuleRegistry mModuleRegistry;
   private UIManager mUIManager;
+  private String mPromptMessage = "";
+  private String mCancelLabel = "";
+  private boolean mDisableDeviceFallback = false;
```

Comment:
I believe that we don't need those variables as members of this class. We only use them in the `authenticateAsync` method. So we can move them there.